CheatSheets/markdown-cheatsheet.md

```diff
@@ -7,15 +7,28 @@
 
 ## Basic Syntax
 
-| Command                                 | Description     |
-| --------------------------------------- | --------------- |
-| `**bold text**` <br /> `__bold text__`  | Bold            |
-| `*italic text*` <br /> `_italic text_`  | Italic          |
-| `# h1` <br /> `## h2` <br /> `### h3`   | # Heading 1  <br /> ## Heading 2 <br /> ### Heading 3     |
-| `[hello google](http://www.google.com)` | Import link     |
-| `- first` <br /> `- second`             | Unordered list  |
-| `1. first` <br /> `2. Second`           | Ordered List    |
-| `---`                                   | Horizontal line |
-| `![alt text](image.jpg)`                | Import Image    |
+| Command                                 | Description                                          |
+| --------------------------------------- | ---------------------------------------------------- |
+| `**bold text**` <br /> `__bold text__`  | Bold                                                 |
+| `*italic text*` <br /> `_italic text_`  | Italic                                               |
+| `# h1` <br /> `## h2` <br /> `### h3`   | # Heading 1 <br /> ## Heading 2 <br /> ### Heading 3 |
+| `[hello google](http://www.google.com)` | Import link                                          |
+| `- first` <br /> `- second`             | Unordered list                                       |
+| `1. first` <br /> `2. Second`           | Ordered List                                         |
+| `---`                                   | Horizontal line                                      |
+| `![alt text](image.jpg)`                | Import Image                                         |
 
+## Extended Syntax
 
+| Command                                                                                                   | Description       |
+| --------------------------------------------------------------------------------------------------------- | ----------------- |
+| `X^2^`                                                                                                    | Superscript       |
+| `H~2~O`                                                                                                   | Subscript         |
+| `I need to highlight these ==very important words==.`                                                     | ==Highlight==     |
```

Comment:
These lines of code do not work on my markdown file. Could you check it in your own `markdown.md` file. Please, notify me if it works or not.